tests/parse_shebang_test.py

```diff
@@ -91,6 +91,13 @@ def test_normexe_does_not_exist_sep():
     assert excinfo.value.args == ('Executable `./i-dont-exist-lol` not found',)
 
 
+def test_normexe_not_executable(tmpdir):
```

Comment:
you'll need `@pytest.mark.xfail(os.name == 'nt', reason='posix only')  # pragma: windows no cover`

sorry forgot about the test-skipping part